Eth_converter.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+contract PriceConsumerV3 {
+
+    AggregatorV3Interface internal priceFeed_eth_usd;
+    AggregatorV3Interface internal priceFeed_btc_eth;
+
+    /**
+     * Network: Kovan
```

Comment:
as noted here, these contract addresses are for the Kovan Ethereum testnet. This isn't relevant for Celo.